core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -676,6 +676,7 @@ private Query showCreateSchema(ShowCreate node)
             Optional<PrincipalSpecification> owner = metadata.getSchemaOwner(session, schemaName).map(MetadataUtil::createPrincipal);
 
             CreateSchema createSchema = new CreateSchema(
+                    node.getLocation().orElseThrow(),
```

Comment:
This `node` (ShowCreate) should always have a location:
https://github.com/trinodb/trino/blob/9ca356eff140244ab9a723e82217751ddf9f7c2b/core/trino-parser/src/main/java/io/trino/sql/tree/ShowCreate.java#L41-L43

Reply:
can we change location in the follow-up PR to be non-optional?

Reply:
The follow-up won't happen soon because the method exists in Node interface.